src/spikeinterface/core/base.py

```diff
@@ -431,10 +435,20 @@ def to_dict(
             # include only main properties
             dump_dict["properties"] = {k: self._properties.get(k, None) for k in self._main_properties}
 
-        if relative_to is not None:
+        if relative_to is None:
+            dump_dict["relative_paths"] = False
+        else:
             relative_to = Path(relative_to).resolve().absolute()
             assert relative_to.is_dir(), "'relative_to' must be an existing directory"
-            dump_dict = _make_paths_relative(dump_dict, relative_to)
+
+            if check_paths_relative(dump_dict, relative_to):
+                dump_dict["relative_paths"] = True
+                dump_dict = make_paths_relative(dump_dict, relative_to)
+            else:
+                dump_dict["relative_paths"] = False
+                # @alessio @ramon @zach: is this warning wanted or not ?
+                warnings.warn("Try to BaseExtractor.to_dict() using relative_to but there is no common folder")
```

Comment:
My question here is that `relative_to` is internal to SpikeInterface machinery right? So as a standard user will this mean anything to mean? When I (being the user that is) use SI don't really think about the fact that `to_dict` is being called to save stuff. It would be helpful to me (Zach), because I'm curious to know when the machinery fails, but I worry this is confusing/ will lead to issues on the tracker...

Reply:
I agree this is why I wanted to open the discussion. warning will be more annoying than informative.

Reply:
I removed the warning.